diff --git a/profiles/xeditor/libraries/ckeditor/config.js b/profiles/xeditor/libraries/ckeditor/config.js
index ea4ff0d..3ea651e 100644
--- a/profiles/xeditor/libraries/ckeditor/config.js
+++ b/profiles/xeditor/libraries/ckeditor/config.js
@@ -4,7 +4,11 @@
  */
 
 CKEDITOR.editorConfig = function( config ) {
-	// Define changes to default configuration here. For example:
-	// config.language = 'fr';
-	// config.uiColor = '#AADC6E';
+  // Define changes to default configuration here. For example:
+  // config.language = 'fr';
+  // config.uiColor = '#AADC6E';
+
+  var custom_plugin = '/profiles/xeditor/modules/xeditor_core/ckeditor/';
+  config.extraPlugins = ['selection'];
+  CKEDITOR.plugins.addExternal('selection', custom_plugin + 'selection/');
 };
